ts/Dashboards/Components/ConnectorHandler.ts

```diff
@@ -64,6 +64,11 @@ class ConnectorHandler {
      * The id of the connector configuration in the data pool of the dashboard.
      */
     public connectorId?: string;
+    /**
+     * Reference to the specific connector data table.
+     */
+    public dataTableKey?: string;
```

Comment:
The same property is in the component class. IMO one should be enough on the component level.

Reply:
I agree, removed from the connector handler level, thanks!